File: challenges/codeeval/easy/192/juancho056.py
import sys

test_cases = open("input.txt", 'r')
for test in test_cases:
    
    dat = map(int, test.split())
    string = ""
    if dat[1] > dat[3]:
        string += "S"
    elif dat[1] < dat[3]:
        string += "N"
    if dat[0] > dat[2]:
        string += "W"
    elif dat[0] < dat[2]:
        string += "E"
    if dat[0] == dat[2] and dat[1] == dat[3]:
        string += "here"
    print string

File: challenges/codeeval/easy/019/juancho056.rb
#Bit Position
File.open("test.txt").each_line do |line|
    n,p1,p2= line.split(",").map(&:to_i) #The map method takes an enumerable object and a block, and runs the block for each element
    num1,num2=[p1,p2].map{|p|(n>>p-1)&1}
    puts num1==num2
end
